### PR TITLE
Add contacts page

### LINKED ISSUE #22: Contact Page
Create sections required for the contact page.

### Tasks:

**Sections**

- [ ] Want to get in touch?
- [ ] Social media media icon links
- [ ] Competitions & Projects
- [ ] For sponsorships
- [ ] Contact us form
- [ ] FAQ

**Components**

- [ ] Contact us form

### PR DESCRIPTION
**Pull request by jsun969**
Partial implementation of #22

### CODE REVIEW

**Comment by jsun969** [comment]
https://github.com/compsci-adl/website/blob/e15f232237824171beed0bca24b5ba99027e6d2a/src/app/contact/Form.tsx#L9-L12

Need backend here.

File: src/app/contact/Contact.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import Link from '@/components/Link';
+import type { Colour } from '@/constants/colours';
+import { LINKS } from '@/data/links';
+import Paragraph from '../../components/Paragraph';
```
Comment (rayokamoto) [inline src/app/contact/Contact.tsx:4]: Change this to `@/components/Paragraph`

File: src/app/contact/Sponsorship.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import FancyRectangle from '@/components/FancyRectangle';
+import Image from 'next/image';
+import Paragraph from '../../components/Paragraph';
```
Comment (rayokamoto) [inline src/app/contact/Sponsorship.tsx:3]: change to `@/components/Paragraph`

File: src/app/contact/page.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import Title from '../../components/Title';
```
Comment (rayokamoto) [inline src/app/contact/page.tsx:1]: Use `@/components/Title`

**Review by rayokamoto — CHANGES_REQUESTED** [review]

**Comment by rayokamoto** [comment]
I wouldn't say this PR fully resolves #22 so although we can merge this, #22 should still be marked as "in progress" as the backend and the email integration does not yet exist.

**Review by phoenixpereira — APPROVED** [review]

**Review by rayokamoto — APPROVED** [review]
Good to go now

**Pull request merged**